openquakeplatform/monitoring/views.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2019, GEM Foundation.
+#
+# This program is free software: you can redistribute it and/or modify
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import urllib2
+import json
+from django.http import HttpResponse
+import psycopg2
+from django.conf import settings
+
+
+def monitoring(request):
+
+    data = []
+
+    GS_LOCATION = settings.GEOSERVER_LOCATION
+
+    title = ['Engine server', 'Geoserver']
+    url = ['http://localhost:8800/v1/engine_version', GS_LOCATION]
+
+    # check Geoserver and Engine server
+    for a_title, a_url in zip(title, url):
+        item = {}
+        try:
+            urllib2.urlopen(a_url, timeout=1)
+            item['Title'] = a_title
+            item['Url'] = a_url
+            item['Status'] = 'OK'
+        except urllib2.URLError, e:
```

Comment:
`URLError` does not report HTTP status errors (like `500` or `404`) but just `IOError` socket errors (endpoint isn't listening)

To be able to catch HTTP errors you should also catch `HTTPError`, I guess

@nastasi-oq do you confirm=